robottelo/cli/proxy.py

```diff
@@ -34,6 +35,7 @@ class SSHTunnelError(Exception):
 @contextlib.contextmanager
 def default_url_on_new_port(oldport, newport):
```

Comment:
We definitely need a better name for this context manager and other refactors on this. But we can take care of it later.